devyiitestdrive/protected/views/timezone/timezone.php

```diff
@@ -5,5 +5,7 @@
  * Date: 27/10/2020
  * Time: 12:10 PM
  */
-
-echo abs(1-1);
\ No newline at end of file
+$renderPageTimeStamp = new DateTime();
+echo "ResponseTimeStamp = " . $timeZoneResponse["timeStamp"] . "</br>";
```

Comment:
TIP: please be aware that this line can possibly be a vector for cross-site scripting